Flash Chat iOS13/Views/MessageCell.swift

```diff
@@ -16,19 +16,17 @@ class MessageCell: UITableViewCell {
 
 	@IBOutlet weak var senderLabel: UILabel?
 
+	@IBOutlet weak var dateTimeLabel: UILabel?
+
 	@IBOutlet weak var leftImageView: UIImageView?
 	
 	@IBOutlet weak var rightImageView: UIImageView?
 
     override func awakeFromNib() {
         super.awakeFromNib()
         // Initialization code
+		// Don't update cell contents here
```

Comment:
❓ Just curious, know how this comment wound up here?

Reply:
That's to let myself (and others) know not to fill the cell's contents there, as it's done in the UITableView delegate methods in ChatViewController. I suppose a case where you would use awakeFromNib() to fill in the cell's contents would be if you wanted them all to include the same things in addition to their unique contents.